### PR TITLE
IoT Deployments with SD-WAN

### LINKED ISSUE #769: IoT Deployments with SD-WAN
### Brief Summary: 
Describe the what, why, and how of your content idea in 2-5 sentences.

Internet of things (IoT) devices require adequate network resources once they are up and running. If left unchecked, IoT devices can place a tremendous strain on a network and lead to a host of performance issues. SD-WAN comes in to support advanced IoT deployments. 
### Key Takeaways: 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?

1. What is SD-WAN
2. SD-WAN vs. traditional WAN 
3. SD-WAN as IoT enabler 

### References: 
Please list links to any published content/research that you intend to use to support/guide this article. (If none, please indicate 'N/A'.)
N/A

### PR DESCRIPTION
**Pull request by EKahuha**
this fixes #769

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by hectorkambow** [comment]
Plagiarism check done

File: articles/iot-deployments-with-sd-wan/index.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+The introduction of SD-WAN changed how a wide area network is managed and deployed. This technology has evolved together with the internet of things (IoT) to become two widely accepted technologies globally. IoT introduces the power of the internet, data analytics, and data processing to the real world of physical devices.
+
+However, various challenges hold the IoT back, including inadequate network resources, security, and lack of interoperability in IoT devices. SD-WAN comes to solve some of the primary issues that IoT is facing.
+
+This article will discuss how SD-WAN ensures successful IoT deployments and how it compares to the traditional wide area network (WAN).
```
Comment (lalith1403) [inline articles/iot-deployments-with-sd-wan/index.md:5]: Add a **Keywords** section. I observe you have mentioned many links. As a reader, I wouldn't be clicking on all the links. After some time, my attention would be fractured. Ensure the reader does not have to refer to many links.  
Explain:
- Overlay networks
- tunneling technology
- MPLS
- data packet
- E2EE
- VPN
- firewalls
and all the major concepts linked that add to the article.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Excellent article. An additional section on keywords and its explanations should make this ready for final review.

**Comment by EKahuha** [comment]
@lalith1403 
Updated.

**Review by lalith1403 — APPROVED** [review]

**Pull request merged**